app/subscriber/src/features/router/AppRouter.tsx

```diff
@@ -54,6 +56,15 @@ export const AppRouter: React.FC<IAppRouter> = () => {
             <PrivateRoute claims={Claim.subscriber} element={<ManageFolder />}></PrivateRoute>
           }
         />
+        <Route
+          path="report/instances/:id/preview"
```

Comment:
Subscribers won't be previewing, they will view an actual report instance output.